src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -109,6 +110,10 @@ module TacoDependencyInstaller {
                 telemetry.step("promptUserBeforeInstall");
                 return this.promptUserBeforeInstall()
                     .then(function (acceptedPrompt: boolean): Q.Promise<number> {
+                        logger.logLine();
+                        loggerHelper.logSeparatorLine();
```

Comment:
This separator used to be logged after we returned from prompting the user; it had the consequence of not being printed when we didn't prompt, so I moved it to a common code path